### PR TITLE
Remove py2 compat

### PR DESCRIPTION
**Pull request by SoapGentoo**

### CODE REVIEW

**Comment by SoapGentoo** [comment]
These are all leaf packages, so should not impact any other packages

File: dev-python/embedly/embedly-0.5.0-r1.ebuild [hunk 1]
```diff
@@ -19,8 +19,7 @@ KEYWORDS="amd64 x86"
 IUSE="test"
```
Comment (mgorny) [inline dev-python/embedly/embedly-0.5.0-r1.ebuild]: Doesn't seem to be used anymore.

File: dev-python/imapclient/imapclient-2.1.0.ebuild [hunk 2]
```diff
@@ -20,8 +20,7 @@ RESTRICT="!test? ( test )"
 RDEPEND="dev-python/six[${PYTHON_USEDEP}]"
 DEPEND="${RDEPEND}
 	dev-python/setuptools[${PYTHON_USEDEP}]
-	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
-	test? ( $(python_gen_cond_dep 'dev-python/mock[${PYTHON_USEDEP}]' -2) )"
```
Comment (mgorny) [inline dev-python/imapclient/imapclient-2.1.0.ebuild]: Likewise.

File: dev-python/imapclient/imapclient-0.13.ebuild [hunk 3]
```diff
@@ -22,8 +22,7 @@ RESTRICT="!test? ( test )"
 
 RDEPEND="dev-python/six[${PYTHON_USEDEP}]"
 DEPEND="${RDEPEND}
-	dev-python/setuptools[${PYTHON_USEDEP}]
-	test? ( $(python_gen_cond_dep 'dev-python/mock[${PYTHON_USEDEP}]' python2_7 pypy) )"
```
Comment (mgorny) [inline dev-python/imapclient/imapclient-0.13.ebuild]: And here.

File: dev-python/hvac/hvac-0.2.17.ebuild [hunk 4]
```diff
@@ -2,11 +2,11 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=( python{2_7,3_5,3_6} )
+PYTHON_COMPAT=( python3_{5,6} )
 
 inherit distutils-r1
 
-DESCRIPTION="Python 2/3 client for HashiCorp Vault"
+DESCRIPTION="Python 3 client for HashiCorp Vault"
```
Comment (mgorny) [inline dev-python/hvac/hvac-0.2.17.ebuild]: Is there a point in saying 'Python 3'? I could get if we're copying some upstream description but if we remove 'Python 2' from it, it makes no sense.

File: dev-python/Kivy/Kivy-1.10.1.ebuild [hunk 5]
```diff
@@ -23,7 +23,7 @@ RDEPEND="
 	media-libs/mesa[gles2]
 	dev-python/pillow[${PYTHON_USEDEP}]
 	cairo? ( dev-python/pycairo[${PYTHON_USEDEP}] )
-	camera? ( media-libs/opencv:0/2.4[$(python_gen_usedep 'python2_7')] )
+	camera? ( media-libs/opencv[${PYTHON_USEDEP}] )
```
Comment (mgorny) [inline dev-python/Kivy/Kivy-1.10.1.ebuild]: Have you verified that the slot dep is unnecessary? If I see something, I'd expect the package to do something really ugly.

File: dev-python/pyproject2setuppy/pyproject2setuppy-2.ebuild [hunk 6]
```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DISTUTILS_USE_SETUPTOOLS=manual
-PYTHON_COMPAT=( python{2_7,3_{5,6,7,8}} pypy{,3} )
+PYTHON_COMPAT=( python3_{5,6,7,8} pypy3 )
```
Comment (mgorny) [inline dev-python/pyproject2setuppy/pyproject2setuppy-2.ebuild]: Keep it here as long as setuptools supports py2. This was just added, and will be used by the eclass.

File: dev-python/pyzor/pyzor-1.0.0-r1.ebuild [hunk 7]
```diff
@@ -16,20 +16,20 @@ SLOT="0"
 KEYWORDS="alpha amd64 hppa ~ia64 ppc ppc64 ~sparc x86 ~amd64-linux ~x86-linux"
 
 IUSE="doc gdbm gevent mysql pyzord redis test"
```
Comment (mgorny) [inline dev-python/pyzor/pyzor-1.0.0-r1.ebuild]: Remove mysql?

File: dev-python/Kivy/Kivy-1.10.1.ebuild (hunk 5 above)
Comment (SoapGentoo) [reply dev-python/Kivy/Kivy-1.10.1.ebuild]: I've just removed the whole `camera?` dep

File: dev-python/pyproject2setuppy/pyproject2setuppy-2.ebuild (hunk 6 above)
Comment (SoapGentoo) [reply dev-python/pyproject2setuppy/pyproject2setuppy-2.ebuild]: Removed from PR

**Pull request closed without merging**